.github/workflows/test.yml

```diff
@@ -0,0 +1,57 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9-dev, pypy3]
+        os: [ubuntu-latest, macos-latest]
+        pytest: [
+          "pytest==3.7.0",
+          "pytest>3.7.0,<4.0.0",
+          "pytest>=4.0.0,<5.0.0",
+          "pytest>=5.0.0,<6.0.0",
+          "pytest>=6.0.0,<7.0.0",
+        ]
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
What about not fetching the entire repository history? (I know it is still not that big though)
```suggestion
      - uses: actions/checkout@v2
         with:
             fetch-depth: 0
```

Reply:
It's actually the other way round :)

> Only a single commit is fetched by default, for the ref/SHA that triggered the workflow. Set `fetch-depth: 0` to fetch all history for all branches and tags.

https://github.com/actions/checkout/

Reply:
😮